skbio/embedding/tests/test_embedding.py

```diff
@@ -69,6 +82,63 @@ def test_assert_length(self):
             SequenceEmbedding(self.emb, self.seq + "A")
 
 
+class SequenceVectorTests(TestCase):
+
+    def setUp(self):
+        # Create some sample SequenceVector objects for testing
+        self.vector1 = np.array([1, 2, 3])
+        self.vector2 = np.array([4, 5, 6])
+        self.vector3 = np.array([7, 8, 9])
+        self.sequence_vectors = [SequenceVector(self.vector1, "ACGT"),
+                                 SequenceVector(self.vector2, "GCTA"),
+                                 SequenceVector(self.vector3, "TTAG")]
+
+    def test_vector(self):
+        # Test if the vector attribute is set correctly
+        for i, vector in enumerate([self.vector1, self.vector2, self.vector3]):
+            self.assertTrue(np.array_equal(self.sequence_vectors[i].vector, vector))
+
+    def test_sequence(self):
+        # Test if the sequence attribute is set correctly
+        for i, sequence in enumerate(["ACGT", "GCTA", "TTAG"]):
+            self.assertEqual(self.sequence_vectors[i].sequence, sequence)
+
+    def test_repr(self):
+        # Test if the __repr__ method returns the correct string
```

Comment:
I dont think that's what this test does. It may be nice if it checked some bare minimum content about the object rather than just the object name, but I don't have strong feelings about that